commands/drive/absoluterelativedrive.py

```diff
@@ -47,9 +51,18 @@ def rotation(self) -> float:
         )
 
     def execute(self) -> None:
-        self.drive.arcadeDriveWithFactors(
```

Comment:
Why did this get added back? I remember it getting removed specifically

Reply:
pathplanner 2024 has the same field symmetry issues of 2023 where you can't just assume red origin and blue origin are equal
this ensures that the robot is always in the perspective of blue origin